src/backend/distributed/executor/multi_executor.c

```diff
@@ -748,6 +748,11 @@ GetObjectTypeString(ObjectType objType)
 {
```

Comment:
I don't understand why we have that function as PG already provides getObjectTypeDescription. As I check we also don't use that function within the PR.